plugins/modules/eks_cluster.py

```diff
@@ -45,6 +45,10 @@
       - present
     default: present
     type: str
+  tags:
+    description:
+      - A dictionary of tags to add the EKS cluster.
+    type: dict    
```

Comment:
Can you add a `version_added: 5.1.0` key here. This is an example: https://github.com/ansible-collections/community.aws/blob/main/plugins/modules/ec2_placement_group.py#L32
This information will be exposed in the module documentation.

Reply:
Added